modules/doc/content/application_development/build_system.md

```diff
@@ -208,3 +208,49 @@ when building MOOSE or a MOOSE-based application.
 
 - `framework/conf_var.mk.in` - This file is where you put your new expansion expressions. Typically these
   are in the form of "@VARAIBLE@".
+
+### Make Install
+
+MOOSE's build system has the ability to create installed binaries suitable for use with the conda package manager, or installation on a shared computing resource such as Sawtooth. The `install` target will copy binary and required libraries to a file tree based on the prefix you configured with (set by the variable `PREFIX` or the `--prefix` argument.
+
+```
+$ cd moose/framework
+$ ./configure --prefix=<installation path>
+$ make
```

Comment:
Missing `cd ../..` back to the app directory.

Reply:
Thanks Yaqi - I will be working on this more as I deprecate the old options and tune this up to meet the needs of Griffin and other requirements that I may have overlooked. This will be useful for me to improve the documentation.